.pylintrc

```diff
@@ -4,4 +4,4 @@ max-module-lines=65536
 good-names=x,y,z,A,t,df,cv
 
 [MESSAGES CONTROL]
-disable=import-outside-toplevel,too-many-arguments,logging-fstring-interpolation,use-maxsplit-arg,fixme
+disable=import-outside-toplevel,too-many-arguments,logging-fstring-interpolation,use-maxsplit-arg,fixme,too-many-locals
```

Comment:
There is an error in line 201 in __main__, saying there are too-many-locals.
I don't know why this issue did not appear in the previous PRs.

Reply:
I don't think we should disable this check. I pushed a fix to your branch.

Reply:
👍